src/common/low_precision_transformations/src/pad.cpp

```diff
@@ -179,9 +178,8 @@ bool PadTransformation::canBeTransformed(const TransformationContext& context, s
         return false;
     }
 
-    if (hasNegativeIndexes(pad)) {
-        return false;
-    }
+    if (!hasPositiveIndexes(pad))
+        return true;
```

Comment:
Is it ok to return true here? Can Pad be transformed if it doesn't have positive pad values?

Reply:
This code is a fix after @v-Golubev comment "Below we have checks for Constant and Reflect pad modes: they are actual only in case of positive or mixed indexes. If all indexes are negative, we can return true before these checks"

Reply:
dequantization operations check:
```
    const auto dequantization = NetworkHelper::getDequantization(op, defaultPrecisions);
    if (dequantization.empty()) {
        return false;
    }
```
has to be before this condition